src/item.cpp

```diff
@@ -2792,6 +2894,12 @@ bool item::is_watertight_container() const
     return type->container && type->container->watertight && type->container->seals;
 }
 
+bool item::is_item_container() const
+{
+    // only check for armour atm
+    return type->armor && (type->armor->storage > 0) && has_flag("HOLDS_ITEMS");
```

Comment:
Might be good temporally, but eventually if it has storage it will be a container.